dev-libs/openssl/files/gentoo.config-1.0.4

```diff
@@ -95,7 +95,7 @@ linux)
 	#	hppa64*)      machine=parisc64;;
 		hppa*)        machine="generic32 -DB_ENDIAN";;
 		i[0-9]86*|\
-		x86_64*:x86)  machine=x86;;
+		x86_64*:x86)  [[ "x$CC" = "xclang" || "x$CC" = "xclang "* ]] && machine=x86-clang || machine=x86;;
```

Comment:
This will fail if `CC=x86_64-pc-linux-gnu-clang`. Also, this is bash, so we don't need the `"x$CC"` weirdness. I suggest the change below.
```suggestion
		x86_64*:x86)  [[ ${CC} == *clang* ]] && machine=x86-clang || machine=x86;;
```